.github/workflows/pr-commitlint.yml

```diff
@@ -12,13 +12,10 @@ jobs:
           fetch-depth: 0
       - name: Install CommitLint and Dependencies
         run: npm install @commitlint/config-conventional @commitlint/cli
-      - name: Lint Commits
-        run: |
-          first_commit=$(curl ${{ github.event.pull_request.commits_url }} 2>/dev/null | jq '.[0].sha' | sed 's/"//g')
-          last_commit=HEAD^2 # don't lint the merge commit
-          npx commitlint --from $first_commit~1 --to $last_commit -V
-      - name: Lint Pull Request
-        env:
-          TITLE: ${{ github.event.pull_request.title }}
-          BODY: ${{ github.event.pull_request.body }}
-        run: export NL=; printenv TITLE NL BODY | npx commitlint -V
+      - name: Ensure follows conventional commits
```

Comment:
I'm testing this on CI so please don't merge unless this comment has been resolved.

Reply:
This doesn't lint the PR title/body?

Reply:
Does this allow commits with "Merge 'develop' into 'xxxxx'"? A few PR's got merged with those merged, which although not a big deal, also doesn't look great, at least to me.

Reply:
You're right, I realized this over the weekend -- It does not in fact check the new working commit. :sweat: